lib/jwt_sessions.rb

```diff
@@ -89,8 +91,17 @@ def algorithm
     @algorithm ||= DEFAULT_ALGORITHM
   end
 
+  def token_store=(args)
+    @token_store =
+      if args.is_a?(JWTSessions::StoreAdapters::AbstractStoreAdapter)
+        args
+      elsif args.is_a?(Array)
+        JWTSessions::StoreAdapters::AbstractStoreAdapter.instance_by_symbol(args.first, args.last)
+      end
```

Comment:
In your tests this method is called as 
```ruby
JWTSessions.store_adapter = :redis
```
Which is an incorrect method usage in this implementation.